### PR TITLE
Address some more defects.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: common/rdm/AckTimerResponder.cpp [hunk 1]
```diff
@@ -220,6 +220,7 @@ const RDMResponse *AckTimerResponder::ResponseFromQueuedMessage(
           queued_response->ParamDataSize());
       break;
     case RDMCommand::SET_COMMAND_RESPONSE:
+      // coverity(SWAPPED_ARGUMENTS)
       return new RDMSetResponse(
```
Comment (peternewman) [inline common/rdm/AckTimerResponder.cpp]: I was looking at this the other day. Shouldn't we create an RDMSetResponse constructor that takes a SetRequest and does half the leg work for you?

**Comment by peternewman** [comment]
Fine aside from the comment.

File: common/rdm/AckTimerResponder.cpp (hunk 1 above)
Comment (nomis52) [inline common/rdm/AckTimerResponder.cpp]: Yeah, that would be reasonable.

Comment (peternewman) [inline common/rdm/AckTimerResponder.cpp]: Well volunteered :)

Comment (peternewman) [inline common/rdm/AckTimerResponder.cpp]: Done! :-1:

**Pull request merged**